### ISSUE TITLE
Remaining tasks for release

### ISSUE THREAD

**Issue by rocky** [Mathics3/pymathics-natlang#12]
- [ ] Add nltk, spacy, pyenchant or pycountry + WMA links (see below for format)
- [ ] Split code into WMA sections: 
   * Text-Level Normalization (TextSentences, TextWords, DeleleteStopWords)
   * Mophological & Lingusitics Normalization (WordStem, DictionaryLookup, DictionaryWordQ, SpellingCorrectionList) 
   * Word List Normalization (WordWordFrequency, WordFrequencyData)
   * Word Manipulation (Pluralize, WordData)
   * Language Translation (LanguageIdentiy)
   * TextAnalysis (WordCount)
 - [ ] Add summary text for builtins

Doc title guide.  We can use use pymathics.graph as an example to compare against.

If there is a wikipedia entry that goes first. See AdjacencyList for an example.

It may be that only a part of the Wikipedia entry is available. See DirectedEdge for an example. 

If there is no wikipedia mention, it is okay to give some free title. EdgeDelete is an example. 

Or you can omit the title altogether. RandomGraph is an Example.

In general we go with the Wikipedia name rather than the WMA for the title. And this include symbolic parameter names. CompleteKaryTree is an example

When the only thing we have is a WMA link we add "link" to the title. EdgeList is an example .

For Summary text, we start with an active verb.  If you look at the section that the summary appears it is nice to use the same verb for similar kinds of things. For example we may "list" builtins that end with "List" (EdgeList, VertexList" but, "find" builtins wiht "Index" at the end of the name "EdgeIndex", "VertexIndex".

**Comment by mmatera** [comment]
Thanks for the detailed guidelines!

**Comment by rocky** [comment]
> Thanks for the detailed guidelines!

You are most welcome, and I regret not having done this sooner. Eventually it will make its way into the developer guide. (Feel free to add it if you want.)

**Comment by mmatera** [comment]
I started with the splitting. I hope to have a PR by the evening.

**Comment by mmatera** [comment]
If someone wants to work in parallel, the best would be to organize urls and tests in a separated file, and then put all together after the splitting be completed.

**Comment by rocky** [comment]
I am not going to have much time before the next weekend.

**Comment by mmatera** [comment]
I guess we are ready with this. Is there something else to do before the release?

**Comment by rocky** [comment]
I don't think so for pymathics.natlang.  And the issues for pymathics.graph I think can be deferred. For a 6.0.1 release.

Any reduction of the cross references in building the LaTeX PDF would be awesome though!
